shared/actions/chat2/index.js

```diff
@@ -2064,8 +2077,20 @@ const navigateToInbox = (state, action, logger) => {
 // (which doesn't count as valid).
 //
 const navigateToThreadRoute = conversationIDKey => {
+  let replace = false
+
+  const visible = Router2Constants.getVisibleScreen()
+  // looking at the pending screen?
+  if (
+    visible?.routeName === 'chatConversation' &&
+    visible?.params?.conversationIDKey === Constants.pendingWaitingConversationIDKey
+  ) {
+    replace = true
+  }
```

Comment:
oh, a back button fix here regressed with deprecating nav1
https://github.com/keybase/client/pull/17439/files#diff-62d6b6a8baeb0bb4a699ed1965bcbf00R2110

Reply:
fixed and pushed tnx